daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -37,7 +37,9 @@ var (
 	// We retry device removal so many a times that even error messages
 	// will fill up console during normal operation. So only log Fatal
 	// messages by default.
-	DMLogLevel int = devicemapper.LogLevelFatal
+	DMLogLevel                   int  = devicemapper.LogLevelFatal
+	DriverDeferredRemovalSupport bool = false
+	EnableDeferredRemoval        bool = false
```

Comment:
I agree it's something we will want by default, but playing it conservative
to allow it to be enabled. Then once proven, we can make it default.
On May 4, 2015 12:59, "Brian Goff" <notifications@github.com> wrote:

> In daemon/graphdriver/devmapper/deviceset.go
> <https://github.com/docker/docker/pull/12190#discussion_r29617251>:
>
> > @@ -37,7 +37,9 @@ var (
> >  	// We retry device removal so many a times that even error messages
> >  	// will fill up console during normal operation. So only log Fatal
> >  	// messages by default.
> > -	DMLogLevel int = devicemapper.LogLevelFatal
> > +	DMLogLevel                   int  = devicemapper.LogLevelFatal
> > +	DriverDeferredRemovalSupport bool = false
> > +	EnableDeferredRemoval        bool = false
>
> Seems like a good idea to make this true by default? Like, this should be
> the expected behavior... but I guess we don't want to change people's envs
> too much.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12190/files#r29617251>.
>